### PR TITLE
feat: bump bifrost tag

### PR DESCRIPTION
**Pull request by jackhodgkiss**

### CODE REVIEW

**Comment by Alex-Welsh** [comment]
If this fixes something important, it would be nice to include a release note

**Comment by mnasiadka** [comment]
What about the centos one?

**Comment by jackhodgkiss** [comment]
> What about the centos one?

I built this for an ubuntu deployment I had no reason to build a centos version. I can do if you feel it would be best.

**Comment by mnasiadka** [comment]
> > What about the centos one?
> 
> I built this for an ubuntu deployment I had no reason to build a centos version. I can do if you feel it would be best.

I think a good reason is that centos image is as old as the ubuntu one - so carries the same issue - can you please rebuild that as well and as Alex mentioned - it would be good to include a reno stating that it fixes an important issue in Ironic database migrations on upgrade

**Comment by markgoddard** [comment]
Let's get this one polished off.

**Comment by Alex-Welsh** [comment]
@jackhodgkiss is this still needed? I'm mentoring a new starter from next Monday and if we still need this I could use it as a teaching aid for how release train and our workflows work if we refresh the container and add a release note

File: releasenotes/notes/bump-bifrost-xena-c0c65bd823ed15fa.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Rebuild and bump the Bifrost container for Xena to include fix for
+    `Error while running update_to_latest_versions: ''BIOSSetting''
+    object has no attribute` during `Ironic database migrations on upgrade`
```
Comment (Alex-Welsh) [inline releasenotes/notes/bump-bifrost-xena-c0c65bd823ed15fa.yaml]: Newline EOF

Comment (jackhodgkiss) [reply releasenotes/notes/bump-bifrost-xena-c0c65bd823ed15fa.yaml]: Done

Comment (Alex-Welsh) [reply releasenotes/notes/bump-bifrost-xena-c0c65bd823ed15fa.yaml]: > Done

are you sure? https://github.com/stackhpc/stackhpc-kayobe-config/compare/8546ab49632989292faea475a82a38d4e42423a9..49cf122387fc55d893a6ee2d68e096088aa68c81

Comment (jackhodgkiss) [reply releasenotes/notes/bump-bifrost-xena-c0c65bd823ed15fa.yaml]: Apologies. I had --amended the commit without staging the changes.

**Review by Alex-Welsh — APPROVED** [review]

**Review by markgoddard — APPROVED** [review]

**Comment by markgoddard** [comment]
Have the images been promoted?

**Comment by jackhodgkiss** [comment]
> Have the images been promoted?

Have just checked with skopeo and both the centos and ubuntu image are present `ark.stackhpc.com/stackhpc/`

**Pull request merged**